tests/serverspec/default.yml

```diff
@@ -67,8 +67,8 @@
   roles:
     - ansible-role-virtual-interface
   vars:
-    virtual_interface:
-      gre0:
+    virtual_interfaces:
```

Comment:
Even if I keep variable name,  actual value is changed from `dict` to `list`. So this isn't backward compatible anyway.